src/mame/includes/cinemat.h

```diff
@@ -88,6 +93,18 @@ class cinemat_state : public driver_device
 	void barrier(machine_config &config);
 	void armora(machine_config &config);
 
+	template<int _N>
```

Comment:
All names beginning with an underscore followed by an uppercase letter are reserved.  Please don’t use them.  Use `LlamaCase` for template parameter names.